### PR TITLE
Store text with emojis using unicode escape.

### PR DESCRIPTION
**Pull request by jiyoung-an**
## Summary
- not converting unicode results in an error during signature generation.
- so, store text with emojis using unicode escape.
  - "How are you today? 😊🌟🌈 "
  - => "How are you today? \\U0001f60a\\U0001f31f\\U0001f308"
- normalize text first and then truncate it.

### CODE REVIEW

File: ainft/utils.py [hunk 1]
```diff
@@ -35,6 +35,14 @@ def now() -> float:
     return datetime.now().timestamp()
 
 
+def truncate_text(text: str, max_length: int) -> str:
+    if max_length <= 0:
+        raise ValueError("max_length must be greater than 0.")
+    if len(text) > max_length:
+        return text[: max_length - 3] + "..."
```
Comment (platfowner) [inline ainft/utils.py:48]: suggestion) how about adding some tests that this works well with unicode strings?

**Review by platfowner — APPROVED** [review]
LGTM with some comments.
Thanks!

File: ainft/utils.py (hunk 1 above)
Comment (jiyoung-an) [reply ainft/utils.py:48]: @platfowner Thanks to your help, I found truncation failed with combined characters (e.g. é, ñ). 
So I now NFKD normalize first and then truncate text!

**Comment by jiyoung-an** [comment]
Thank you for review @platfowner! now merging..
